docs/lab/assets/frontend/React_Tables.md

```diff
@@ -36,6 +36,8 @@ Another coffee script example demonstrates the usage of a custom render function
 ```
 This function checks for `null` first and if `field` is not null it adds ` %` to the `field` value.
 
+Reminder: The render function is not ment to be used for calculations. Since the render function is allowed to return all kinds of elements, including further HTML or React elements it will not be evaluated for sorting. Its only purpose it to have a more user friendly graphical representation of the value. 
```

Comment:
looks like a typo here "is not ment" -> "is not meant"

Reply:
corrected, thanks